File: cs/1.tutorial/ie.cs
using System;

public class IfElse
{
  public static void Main(string[] args)
  {
    int x = 20, y = 18; // declaring our variabnles to be used
    if (x > y) { // if x is greater than y, then we will execute the next code block
      Console.WriteLine("x is greater than y.");
    } else if (x < y) { // if x is not greater then y, we will execute this code block
      Console.WriteLine("y is greater than x.");
    } else { // if none of the 'if' statements or 'else if' statements are true, we will execute this code block
      Console.WriteLine("something went wrong.");
    }
  }
}

File: c/1.tutorial/ie.c
#include <stdio.h>
#include <stdbool.h>

int main() {
  int x = 10, y = 20;
  if (x < y) {
    printf("y is greater than x");
  } else if (x > y) {
    printf("x is greater than y");
  } else {
    printf("something went wrong!");
  }
  return 0;
}
